### PR TITLE
Add error handling for QuickStart Playground

### PR DESCRIPTION
**Pull request by manodasanW**
## Summary of the pull request

- Added an error window for any errors during project generation
- Made the cancel button during extension initialization go back to the extension selection rather than continuing.

## Detailed description of the pull request / Additional comments

image

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml [hunk 1]
```diff
@@ -220,6 +220,16 @@
             </Grid>
 
             <!--Bottom half of UI-->
+            <Grid Grid.Row="2" Visibility="{x:Bind ViewModel.IsErrorViewVisible, Mode=OneWay}">
+                <StackPanel
```
Comment (krschau) [inline tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml:224]: The icon above the text looks a little weird to me, should it move to be next to the TextBlocks? Also, suggest using TextWrapping="WrapWholeWords"

Comment (manodasanW) [reply tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml:224]: That is an interesting thought about the icon placement.  Right now, it is place where design has suggested, but I can bring this up later with them and address it in a future PR if they don't disagree with it.  Will address WrapWholeWords.

Comment (adrastogi) [reply tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml:224]: Does the guidance on wrapping whole words apply generally? (We have a few other uses in TextBoxes across this XAML; can / should those be corrected as well?)

Comment (manodasanW) [reply tools/SetupFlow/DevHome.SetupFlow/Views/QuickstartPlaygroundView.xaml:224]: It is not supported with TextBox, but TextBlock does support it.  Addressed throughout: https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.textbox.textwrapping?view=windows-app-sdk-1.5#remarks

**Review by adrastogi — APPROVED** [review]
I tested out the new behaviors and they look good!

**Pull request merged**